code/modules/surgery/advanced/revival.dm

```diff
@@ -26,7 +26,7 @@
 	return TRUE
 
 /datum/surgery_step/revive
-	name = "repair body"
+	name = "electrically stimulate brain"
```

Comment:
Just change it to "revive body", doctor Frankenstein.

Reply:
ooga